beacon-chain/blockchain/receive_block.go

```diff
@@ -181,6 +194,14 @@ func (s *Service) ReceiveBlockNoVerify(ctx context.Context, block *ethpb.BeaconB
 		}
 	}
 
+	// Send notification of the processed block to the state feed.
```

Comment:
We don't need to do this in `ReceiveBlockNoVerify`

Reply:
Do we not?  There was an old HeadUpdated event in here, and this still updates the database with the block, updates head, *etc.* so why wouldn't we need to trigger the event?

Reply:
Added `Verified` flag to differentiate.